tslint.json

```
@@ -84,7 +84,7 @@
     "no-irregular-whitespace": true,
     "no-magic-numbers": false,
     "no-misused-new": true,
-    "no-namespace": [true, "allow-declarations"],
+    "no-namespace": false,
```

```suggestion
    "no-namespace": [true, "allow-declarations"],
```
namespace 就是 namespace，不要当模块使用。